cql3/statements/cf_prop_defs.cc

```diff
@@ -162,6 +176,17 @@ void cf_prop_defs::validate(const data_dictionary::database db, sstring ks_name,
     }
 
     speculative_retry::from_sstring(get_string(KW_SPECULATIVE_RETRY, speculative_retry(speculative_retry::type::NONE, 0).to_sstring()));
+
+    auto tablet_options_map = get_tablet_options();
+    if (!tablet_options_map.empty()) {
```

Comment:
We should reject `with tablets = {}` too when the feature is not enabled, since it will emit mutations which null out the map.

Reply:
Fixed in v13 ([c8b1548](https://github.com/scylladb/scylladb/commit/c8b154835c45cc27b8f0ec87764a9c9d62f455f3))